src/domain/organisation/organisation.service.ts

```diff
@@ -35,6 +35,16 @@ export class OrganisationService {
     return organisation;
   }
 
+  async getOrganisationByID(organisationID: number): Promise<IOrganisation> {
+    //const t1 = performance.now()
+    const organisation = await Organisation.findOne({
+      where: [{ id: organisationID }],
+    });
+    if (!organisation)
+      throw new Error(`Unable to find challenge with ID: ${organisationID}`);
```

Comment:
Should be: "Unable to find **organisation** with ID: ${organisationID}`"

Reply:
To be fixed